lib/imgix/client.rb

```diff
@@ -60,6 +52,27 @@ def prefix
 
     private
 
+    def create_request(endpoint, img_url)
+      req = Net::HTTP::Post.new(endpoint.path)
+      req["Content-Type"] = "application/json"
+      req["Authorization"] = "Bearer #{@api_key}"
+      req["User-Agent"] = "imgix #{@library}-#{@version}"
+      req.body = json_data(img_url)
+
+      req
+    end
+
+    def json_data(url)
```

Comment:
This could probably be better named `purge_json_data` or `gen_purge_json_body`

Reply:
Yeah, hmmm, what about `json_body_from_url`? I just don't want to give the impression that `json_body` does anything other than return a json string (for me, this kind of excludes `purge_json_data`). I guess we can also call this `mock_json_data`...

Reply:
Right, yeah fair call on not wanting to give that impression. I just thought `json_data` was a bit too unspecific since this was specific to purging. What did you think about `gen_purge_json`? Or any of `gen_purge_json_body`, `create_purge_json`, `gen_purge_data`, `create_purge_data`?